app/views/incidental_types/_form.html.erb

```diff
@@ -1,45 +1,36 @@
 <%= form_for(@incidental_type) do |f| %>
   <div class="form-group col-xs-12">
-    <%= f.label :name, 'Name', class: 'control-label col-xs-2' %>
+    <%= f.label :name, "Name", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= f.text_field :name, class: 'form-control', required: true %>
+      <%= f.text_field :name, class: "form-control", required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= f.label :description, 'Description', class: 'control-label col-xs-2' %>
+    <%= f.label :description, "Description", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= f.text_field :description, class: 'form-control', required: true %>
+      <%= f.text_field :description, class: "form-control", required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= f.label :base, 'Base Fee', class: 'control-label col-xs-2' %>
+    <%= f.label :base, "Base Fee", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= f.number_field :base, class: 'form-control', min: 0, step: 0.01, required: true %>
+      <%= f.number_field :base, class: "form-control", min: 0, required: true %>
     </div>
   </div>
 
   <div class="form-group col-xs-12">
-    <%= f.label :damage_tracked, 'Damage Tracked?', class: 'control-label col-xs-2' %>
+    <%= f.label :damage_tracked, "Damage Tracked?", class: "control-label col-xs-2" %>
     <div class="col-xs-6">
-      <%= f.check_box :damage_tracked, class: '' %>
+      <%= f.check_box :damage_tracked %>
     </div>
   </div>
 
-<% if false %>
-  <div class="form-group col-xs-12">
-    <%= f.label :rental_ids, 'Rental', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.select :rental_ids, options_from_collection_for_select(@rentals, 'id', 'name', selected: @incidental_type.rentals.try(:ids)), {}, multiple: true, class: "selectpicker form-control" %>
-    </div>
-  </div>
-<% end %>
-
   <div class="form-group col-xs-12">
     <div class="col-xs-offset-2 col-xs-10">
-      <%= f.submit class: "btn btn-primary", data: { confirm: 'Are you sure?' } %>
-      <%= link_to 'Back', :back, class: "btn btn-default" %>
+      <%= f.submit class: "btn btn-primary", data: { confirm: "Are you sure?" } %>
+      <%= link_to "Back", :back, class: "btn btn-default" %>
```

Comment:
My only gripe so far with your changes is this "I need double-quotes" thing, but it's not a blocking concern because it really doesn't matter that much. That being said, it wasn't worth putting effort into. You don't need double-quotes in ruby unless you a) have a single quote within it or b) are doing interpolation of some kind.

Reply:
While I almost always prefer single quotes in code, I like to almost exclusively use double quotes in html. In shared code-bases where everybody has their own style, I'll change everything to double quotes for consistency. It wasn't any effort at all, it was a simple find-replace all (and make sure it doesn't break any embedded quotes situation)

Reply:
Fine, but in that case I'll point out a couple things: 
One, it added a lot of bulk to a PR that was already pretty bulky, and wasn't a functional change. I'd have put it in its own PR.
Two, this isn't HTML in this context--while the document gets turned into HTML, every place you've used double quotes here is Ruby (and necessarily so, because if it were HTML, they'd get read as plain text).
Three: As an organization, we (okay, at least here at Transit, Parking was pretty separate for a while) prefer single quotes unless double-quotes are necessary--because it's easier to find interpolation when scanning a document that way.
Either way, any argument for or against single vs double quotes is pretty weak, so this is not a hill I'm going to die on, do what you like. Haha